services/web/server/src/simcore_service_webserver/projects/_crud_handlers_models.py

```diff
@@ -52,6 +53,30 @@ def search_check_empty_string(cls, v):
             return None
         return v
 
+
+class ProjectListWithJsonStrParams(ProjectListParams):
+    order_by: Json[OrderBy | None] = Field(  # pylint: disable=unsubscriptable-object
```

Comment:
Ohh boy this `Json` field is a pain! I would advise to use a pre validator to transform the str to a json and then use a dict instead.
Let's see if @pcrespov has a better idea.

Reply:
Ye got your point, I put some thinking and testing into it and this was the best approach I found, it's also consistent with resource-usage listing therefore I reuse the same logic and Pydantic model here. What is good with this JSON string type in comparison to your proposal is that Pydantic deals with the validation for me (I do not have to have a custom validator for the same thing) + what is convenient is that it is already embedded in the OAS specs :+1: